include/torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h

```diff
@@ -95,6 +95,16 @@ struct OpBinder {
     return success();
   }
 
+  ParseResult tensorResultTypeAtIndex(Torch::ValueTensorType &typeIdx, int64_t idx) {
+    if (idx >= op->getNumResults())
+      return failure();
+    auto t = toValidTensorType(op->getResult(idx).getType());
+    if (!t)
+      return failure();
+    typeIdx = t;
+    return success();
+  }
+
```

Comment:
This code is already added here: https://github.com/llvm/torch-mlir/commit/80bd093d56c5f7b36c6852fdff05afac0d7c3a00

Reply:
You can create different branches for different PRs.

Reply:
Yes should I start this one in a new branch ? Which would lead to a new PR. I understand that it is a bit cleaner that way, but I created the first PR as an interim step and the LayerNorm depended on that so by the time the first PR was accepted and merged I already moved on with the LayerNorm.